frontend/src/App.tsx

```diff
@@ -5,7 +5,9 @@ import Article from './components/ArticleSection/index';
 
 function App() {
   const ESSENTIALLY = 'Essentially';
-  const [text, setText] = useState<string[]>([]);
+  const [text, setText] = useState<any>({
```

Comment:
we should avoid using any. if it's not a string please make an interface for it

Reply:
done